CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ------
+* Remove --no-optional when using npm to create new extension
```

Comment:
@henrytao-me can you please include the PR #

Reply:
Added! 🙇